electron/anticheat/utils.ts

```diff
@@ -7,18 +8,20 @@ import { AntiCheatInfo } from '../types'
 async function downloadAntiCheatData() {
   if (!isLinux) return
 
-  try {
-    const { data } = await axios.default.get(
-      'https://raw.githubusercontent.com/Starz0r/AreWeAntiCheatYet/HEAD/games.json'
-    )
-    writeFileSync(heroicAnticheatDataPath, JSON.stringify(data, null, 2))
-    logInfo(`AreWeAntiCheatYet data downloaded`, LogPrefix.Backend)
-  } catch (error) {
-    logWarning(
-      `Failed download of AreWeAntiCheatYet data: ${error}`,
-      LogPrefix.Backend
-    )
-  }
+  ipcMain.once('online', async () => {
```

Comment:
we can use this pattern to do things only once the first time we detect connectivity, useful for one-off in-the-background things like this